bench/benchmarks/geojson_rendering.js

```diff
@@ -0,0 +1,127 @@
+
+var Evented = require('../../js/util/evented');
+var util = require('../../js/util/util');
+var formatNumber = require('../format_number');
+
+module.exports = function(options) {
+  var evented = util.extend({}, Evented);
+
+  var totalFPS = 0;
+  var countRuns = 0;
+
+  var runner = function(zoom) {
+    countRuns++;
+    var map = options.createMap({
+      width: 512,
+      height: 512,
+      zoom: zoom
+    });
+
+    runTest(map, function(fps) {
+      totalFPS+=fps;
+      evented.fire('log', {message: 'zoom '+zoom+' - '+formatNumber(fps)+' fps'});
+      map.remove();
+      if (zoom < 22) {
+        runner(zoom+1);
+      }
+      else {
+        evented.fire('end', {message: 'average across zooms - '+formatNumber(totalFPS/countRuns)});
+      }
+    });
+  }
+
+  runner(0);
+
+  return evented;
+}
+
+var featureCollection = {
+  type: 'FeatureCollection',
+  features: []
+}
+
+for (var idx=0; idx<45; idx+=.15) {
+  featureCollection.features.push({
+    type: 'Feature',
+    properties: {},
+    geometry: {
+      coordinates: [[
+        [0-idx, 0],
+        [0, 0-idx],
+        [0+idx, 0],
+        [0, 0+idx],
+        [0-idx, 0]
+      ]],
+      type: 'Polygon'
+    }
+  });
+}
+
+function runTest(map, cb) {
+  var numFrames = 0;
```

Comment:
We should extract `measureFramerate` function from the fps benchmark into a reusable utility. This will ensure our measurement methodology is consistent (and simplify this codepath).